projects/core/components/dialog/dialog.service.ts

```diff
@@ -1,24 +1,20 @@
-import {Injectable} from '@angular/core';
+import {inject, Injectable} from '@angular/core';
 import {AbstractTuiDialogService} from '@taiga-ui/cdk';
-import {TuiDialogOptions} from '@taiga-ui/core/interfaces';
+import type {TuiDialogOptions} from '@taiga-ui/core';
```

Comment:
Looks like this should still be from the `interfaces` entry point.

Reply:
<img width="1314" alt="image" src="https://user-images.githubusercontent.com/12021443/197218620-63e8067e-7413-4deb-a83f-fbfc0a85f372.png">

Reply:
This import apparently was auto-fixed by this rule: @taiga-ui/no-deep-imports. 
So right now, I can't commit this fix. If we leave it like this, there is the `dependency cycle` issue. So should I just `eslint-ignore` the `no-deep-imports` for now?

Reply:
In theory it should work
`import {TuiDialogOptions} from '@taiga-ui/core/interfaces';`

otherwise it's a bug in our eslint plugin

Reply:
@splincode have a look
![image](https://user-images.githubusercontent.com/37042257/197596338-7aeccd46-3622-4321-a9fc-ff433da7e12e.png)

Reply:
However, same thing happens for other files that are already commited. I checked with this other file that was edited today in this PR: https://github.com/Tinkoff/taiga-ui/pull/2955/files and below you can see the same error. However in the PR workflow checks it does not show this error at all

![image](https://user-images.githubusercontent.com/37042257/197598413-9f86c5a6-45a1-4126-8003-e3202b217211.png)

Reply:
Turns out there is actually a bug in the `no-deep-imports.js` plugin. When searching for `currentProject` token, you search in the file absolute path. So, if by any chance, someone like me has the `taiga-ui` library inside a parent directory named `projects`, the script never finds the correct library name :smile: 

You can see in the pictures above that the library was inside `/home/festimprebreza/projects/...`

Reply:
could you fix this bug?

Reply:
@nsbarsukov fyi

Reply:
@splincode sure :+1:

Reply:
@waterplea can you please review the changes. I rebased and squashed directly since they were small changes